docs/data/base/components/tabs/UnstyledTabsIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { Tabs } from '@mui/base/Tabs';
+import { TabsList as BaseTabsList, TabsListProps } from '@mui/base/TabsList';
+import { TabPanel as BaseTabPanel, TabPanelProps } from '@mui/base/TabPanel';
+import { Tab as BaseTab, TabProps } from '@mui/base/Tab';
+
+export default function UnstyledTabsIntroduction() {
+  return (
```

Comment:
We need to have a wrapping div here that will set the 'dark' classname in dark mode.